LOL. This is [fun](https://circleci.com/gh/conda-forge/conda-forge-build-setup-feedstock/77). 😒

```
Using Anaconda API: https://api.anaconda.org
Traceback (most recent call last):
  File "/feedstock_root/ci_support/upload_or_check_non_existence.py", line 124, in <module>
    main()
  File "/feedstock_root/ci_support/upload_or_check_non_existence.py", line 104, in main
    exists = built_distribution_already_exists(cli, meta, owner)
  File "/feedstock_root/ci_support/upload_or_check_non_existence.py", line 24, in built_distribution_already_exists
    fname = bldpkg_path(meta)
TypeError: bldpkg_path() missing 1 required positional argument: 'config'
```

I think we are going to have to do some pinnings in the CI scripts too. It's ugly, but tolerable for now.

--- a/.travis.yml
+++ b/.travis.yml
@@ -28,6 +28,7 @@ install:
       conda update --yes conda
       conda install --yes conda-build jinja2 anaconda-client
       conda config --add channels conda-forge
+      conda install --yes conda-build=1
       
 
 script:

--- a/ci_support/run_docker_build.sh
+++ b/ci_support/run_docker_build.sh
@@ -41,6 +41,7 @@ conda clean --lock
 
 conda update --yes --all
 conda install --yes conda-build
+conda install --yes conda-build=1
 conda info
 
 # Embarking on 1 case(s).

--- a/recipe/run_conda_forge_build_setup_win
+++ b/recipe/run_conda_forge_build_setup_win
@@ -11,7 +11,7 @@ conda config --set add_pip_as_python_dependency false
 conda update -n root --yes --quiet conda
 conda install -n root --yes --quiet jinja2 conda-build anaconda-client
 
-# KLUDGE to work around changes in conda-build 2.0.0
+:: KLUDGE to work around changes in conda-build 2.0.0
 conda install -n root --yes --quiet conda-build=1
 
 conda info

